Makefile

```diff
@@ -257,8 +257,17 @@ CFLAGS += -Wno-cast-function-type
 # Defined Symbol (MACROS)
 CFLAGS += -D__HEAP_SIZE=0
 CFLAGS += -DCONFIG_GPIO_AS_PINRESET
-CFLAGS += -DCONFIG_NFCT_PINS_AS_GPIOS
-CFLAGS += -DSOFTDEVICE_PRESENT
+
+# Skip defining CONFIG_NFCT_PINS_AS_GPIOS if the device uses the NFCT.
+ifneq ($(USES_NFCT),yes)

```

Comment:
would you mind changing it to just `USE_NFCT`, I found it easier to read with all cap.

Reply:
Definitely. I like the way that reads better too.